pkg/server/backend_manager.go

```diff
@@ -38,8 +39,13 @@ var _Backend = new(agent.AgentService_ConnectServer)
 type backend struct {
 	// TODO: this is a multi-writer single-reader pattern, it's tricky to
 	// write it using channel. Let's worry about performance later.
-	mu   sync.Mutex // mu protects conn
-	conn agent.AgentService_ConnectServer
+	mu      sync.Mutex // mu protects conn
+	conn    agent.AgentService_ConnectServer
+	AgentID string
```

Comment:
As far as I can tell, thus does not need to be public. Can we switch it to "agentID"?

Reply:
This was originally intended to be for debugging and I didn't find an easy way to incorporate this without regenerating the proto. Will remove for now.